template/runner-config.tpl

```diff
@@ -65,6 +65,7 @@ listen_address = "${prometheus_listen_address}"
       "amazonec2-iam-instance-profile=%{ if runners_iam_instance_profile_name != "" }${runners_iam_instance_profile_name}%{ else }${runners_instance_profile}%{ endif ~}",
       "amazonec2-root-size=${runners_root_size}",
       "amazonec2-volume-type=${runners_volume_type}",
+      "amazonec2-userdata=%{ if runners_userdata != "" }/etc/gitlab-runner/runners_userdata.tpl%{ endif ~}",
```

Comment:
chore: to be tested without specifying `runners_userdata`

Reply:
Successfully tested. The executors are still running. Nevertheless I had problems with one executor which was not able to update the Ubuntu system. Quite strange. Seems not to be related to this PR.

@npalm Just to be on the safe side: Could you test this as well please?